ui/raidboss/data/06-ew/trial/zeromus-ex.ts

```diff
@@ -246,23 +254,37 @@ const triggerSet: TriggerSet<Data> = {
           }
         }
 
-        if (possibleSafeSpots.length !== 2)
+        if (possibleSafeSpots.length !== 1)
           return output.avoidUnknown!();
 
-        const [safeDir1, safeDir2] = possibleSafeSpots;
-        if (safeDir1 === undefined || safeDir2 === undefined)
+        const [safeDir] = possibleSafeSpots;
+        if (safeDir === undefined)
           return output.avoidUnknown!();
 
-        return output.combo!({ dir1: output[safeDir1]!(), dir2: output[safeDir2]!() });
+        return output[safeDir]!();
       },
       outputStrings: {
-        combo: {
-          en: '${dir1} / ${dir2}',
-        },
         avoidUnknown: {
           en: 'Avoid Line Cleaves',
         },
-        ...Directions.outputStrings16Dir,
+        dirNNE: {
+          en: 'North Wall (NNE/WSW)',
+        },
+        dirNNW: {
+          en: 'North Wall (NNW/WSW)',
```

Comment:
I think this should be "North Wall (NNW/ESE)".

Reply:
Whoops!